test/pkg/consts/const.go

```diff
@@ -7,4 +7,6 @@ const (
 	WPCDeviceType = "E810-XXV-4T"
```

Comment:
Exact device type is E810XXVDA4T. See suggestion below:

```suggestion
 	WPCDeviceType = "E810-XXVDA4T"
```

Reply:
The device type we have is `E810-XXV-4T` do you have `E810-XXVDA4T`? Also that was part of the previous PR, not set in this PR, so we can open another PR to change it or that will be done in a PR that's adding the config yaml you previously suggested.

Reply:
let's do that later then :-)